pkg/uki/upgrade.go

```diff
@@ -121,6 +121,15 @@ func (i *UpgradeAction) installRecovery() error {
 	if err != nil {
 		return err
 	}
+	err = replaceRoleInKey(targetConfPath, "efi", UnassignedArtifactRole, "recovery", i.cfg.Logger)
+	if err != nil {
+		return err
+	}
+
+	err = replaceConfTitle(targetConfPath, "recovery")
```

Comment:
Shouldn't we do this on installation too?

Reply:
Oh I see, you assume that from now on, the menus will [be correct from enki](https://github.com/kairos-io/enki/pull/77/files). Gotcha, all good.